### ISSUE TITLE
Add clock for how much time is left in the break

### ISSUE THREAD

**Issue by junewunder** [BarlowProgramming/redirect#21]
**this is another feature idea for the future**

There's no way to know how much time is left in the break.  We should have a clock or a circle that of some sort to know how much time is left.

Either we can use JS with a canvas or a CSS animation with a time limit (?) I don't know how CSS animations work so I'm not sure if that's a thing.

**Comment by jlam55555** [comment]
My ideas:
- Worse: An analog clock (I'm guessing this was what you were implying)
  - Worse: CSS animations would probably not still continue in the background, and it might not work for longer periods of time. 
  - Better: CSS transforms for the hands of the clock but JS timers. This would make managing the animation or controlling over long periods of time
- Better: A digital clock 
  - Isn't this a more practical idea?

**Comment by junewunder** [comment]
Yes the Worse-Better is what I was thinking. Digital clocks stress me out personally, maybe there could be an option for both, but I think like a circle with a grey area for the amount of time left.  Doesn't have to look too complex.  

**Question:**  Would SVG work best? I believe this is what SVG is for
